config/jobs/kubernetes/sig-release/release-branch-jobs/1.17.yaml

```diff
@@ -1249,9 +1249,11 @@ presubmits:
     spec:
       containers:
       - args:
+        - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
+        - --repo=sigs.k8s.io/kind=7b28ee53357ee77f111d2e7b1b602876dc6d0d4c
```

Comment:
ughhh this job is actually podutils + kubetest, not bootstrap + kubetest. too bad we can't auto validate this